H.Ws/H.W_4.py

```diff
@@ -0,0 +1,236 @@
+import numpy
+import pandas
+import statsmodels.api
+from plotly import express as px
+from plotly import figure_factory as ff
+from plotly import graph_objects as go
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.metrics import confusion_matrix
+from sklearn.model_selection import train_test_split
+
```

Comment:
I think its a good idea to include a main in your python code. Having all your code as global variables introduces a lot of room for bugs in your code. It also debugging large amounts of code a pain in the butt